docs/install/upgrade/upgrade-installation-with-operator.md

```diff
@@ -131,8 +134,9 @@ If the upgrade fails, you can rollback to restore your Knative to the previous v
       name: knative-serving
       namespace: knative-serving
     spec:
-      version: "0.22"
+      version: "<previous-version>"
     ```
+    Where `<previous-version>` is the Knative version that you want to upgrade to.
```

Comment:
```suggestion
    Where `<previous-version>` is the Knative version that you want to downgrade to.
```

Since it's rolling back to a previous version, I read it as doing a downgrade. What do you think about using `downgrade` instead of `upgrade`? Or something in the lines of *"Where `<previous-version>` is the Knative version that you want to rollback to"*.